<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanWindowDialogSetAxis</class>
 <widget class="QMainWindow" name="PanWindowDialogSetAxis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Axis Feature</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxXAxis">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>500</width>
      <height>125</height>
     </rect>
    </property>
    <property name="title">
     <string>X Axis</string>
    </property>
    <widget class="QLabel" name="labelXAxisTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>70</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Axis Title:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditXTitle">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>15</y>
       <width>400</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelXAxisTitleSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>65</y>
       <width>70</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Title Size:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxXTitleSize">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
    </widget>
    <widget class="QFontComboBox" name="fontComboBoxXTitleFont">
     <property name="geometry">
      <rect>
       <x>258</x>
       <y>60</y>
       <width>220</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelXAxisTitleSize_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>65</y>
       <width>70</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Title Font:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelXScaleSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>70</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale Size:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxXScaleSize">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>95</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelXScaleFont">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>70</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale Font:</string>
     </property>
    </widget>
    <widget class="QFontComboBox" name="fontComboBoxXScaleFont">
     <property name="geometry">
      <rect>
       <x>258</x>
       <y>95</y>
       <width>220</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxYAxis">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>500</width>
      <height>125</height>
     </rect>
    </property>
    <property name="title">
     <string>Y Axis</string>
    </property>
    <widget class="QLabel" name="labelYAxisTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>70</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Axis Title:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditYTitle">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>15</y>
       <width>400</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelYAxisTitle_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>65</y>
       <width>70</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Title Size:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxYTitleSize">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
    </widget>
    <widget class="QFontComboBox" name="fontComboBoxYTitleFont">
     <property name="geometry">
      <rect>
       <x>258</x>
       <y>60</y>
       <width>220</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelYAxisTitleFont">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>65</y>
       <width>70</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Title Font:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelXScaleSize_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>70</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale Size:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxYScaleSize">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>95</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
    </widget>
    <widget class="QFontComboBox" name="fontComboBoxYScaleFont">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>95</y>
       <width>220</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelXScaleFont_2">
     <property name="geometry">
      <rect>
       <x>182</x>
       <y>100</y>
       <width>70</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale Font:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonOK">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCancel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
